assets/search/components/SearchResultsBar/SearchResultsTopicRow.tsx

```diff
@@ -58,7 +58,7 @@ export function SearchResultsTopicRow({
         );
     }
 
-    if (get(searchParams, 'navigation.length', 0)) {
+    if ((searchParams.navigation?.length ?? 0) > 0 && ((Object.keys(navigations).length ?? 0) > 0)) {
```

Comment:
`Object.keys` returns an empty array if there are no keys, thus defaulting `length` to zero will never execute